### DISCUSSION TITLE
ImportError: cannot import name 'embedded_io' from 'lava.proc' (unknown location)

### DISCUSSION THREAD

**Discussion by kimdongyeong0** [lava-nc/lava-dl#368] [Q&A]
Hello, this is Dongyeong

I tried to run yolo_kp on Loihi2 chip. When I ran run.py in the yolo_kp, there is error that "ImportError: cannot import name 'embedded_io' from 'lava.proc' (unknown location)". 

I set virtual environment both of lava-nc and lava-dl respectively. What is lava.proc and how can I import them?

**Comment by Yan-Zen9** [comment]
Hi Dongyeong,
Do you find the solution? I meet the same problem. Could you share your method if you find the solution?

**Reply by kimdongyeong0** [reply]
Did you try reinstalling the virtual environment specifically for Loihi, following the official installation guide provided by Intel (available to INRC members)? This virtual environment is different from lava-nc and lava-dl. In my case, I simply reinstalled the virtual environment, and the error disappeared.
